src/python/turicreate/test/test_style_transfer_tf.py

```diff
@@ -0,0 +1,130 @@
+# -*- coding: utf-8 -*-
+# Copyright © 2017 Apple Inc. All rights reserved.
+#
+# Use of this source code is governed by a BSD-3-clause license that can
+# be found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
+from __future__ import print_function as _
+from __future__ import division as _
+from __future__ import absolute_import as _
+import unittest
+import pytest
+from . import util as test_util
+import turicreate as tc
+import tempfile
+import numpy as np
+import platform
+import sys
+import os
+from turicreate.toolkits._main import ToolkitError as _ToolkitError
+from turicreate.toolkits._internal_utils import _raise_error_if_not_sframe, _mac_ver
+import coremltools
+
+from ..toolkits.style_transfer._tf_model_architecture import define_resnet as _define_resnet
+from ..toolkits.style_transfer._tf_model_architecture import define_instance_norm as _define_instance_norm
+
+from ..toolkits.style_transfer._model import InstanceNorm as _InstanceNorm
+from ..toolkits.style_transfer._model import ResidualBlock as _ResidualBlock
+from ..toolkits.style_transfer._model import Transformer as _Transformer
+
+import tensorflow as _tf
+import mxnet as _mx
```

Comment:
Why do we need to import MXNet?

Reply:
Refactored not to import globally.

Reply:
This is because we'd like to validate the difference between the MxNet and TF Implementations.